### ISSUE TITLE
Remove 16 core options from choose_slurmConfig()

### ISSUE THREAD

**Issue by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [remindmodel/remind#1537]
Using 16 over twelve cores does not yield any benefit and is therefore removed from the defaults (#1536).  It would be only consistent to also remove the option from `choose_slurmConfig()`.

https://github.com/remindmodel/remind/blob/0695bd08452d99d940cdf449f4566338fe787968/scripts/start/choose_slurmConfig.R#L15-L34

**Comment by orichters** [comment]
- As `start_bundle_coupled.R` does not use this function but always runs on 13 core for nash runs, also the 13 core options can be removed, I think.
- For coupled runs, [the script identifies itself](https://github.com/remindmodel/remind/blob/develop/start_bundle_coupled.R#L540-L546) how many nodes are advisable (13 for nash mode, 1 for test and negishi).
- Also, we have an `auto` option that checks if a priority slot is free and takes priority then, and short otherwise
- Maybe we can simply restrict the user choice to auto (recommended), priority, standby, short, medium, long, direct?

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
> Maybe we can simply restrict the user choice to auto (recommended), priority, standby, short, medium, long, direct?

In my opinion, "direct" should not be an option.  On the cluster, computation loads should always go to the compute nodes, not the login nodes.  Locally, the entire thing is bypassed anyhow.

**Comment by orichters** [comment]
Yes, fully agree, that part of the code is missing yet. We just have to keep in mind that some files in `config/tests` use their own special slurm settings (particularly `--wait`).

**Comment by orichters** [comment]
As noticed by @robertpietzcker, `make test` currently requires a `priority` slot which can be annoying. We should replace that by `auto`, I think, once that is supported.

**Comment by orichters** [comment]
We might want to allow the user also to select limiting the slurm time, useful in particular in case of upcoming cluster closures...
